pkg/mimirpb/compat.go

```diff
@@ -156,6 +157,42 @@ func FromExemplarProtosToExemplars(es []Exemplar) []exemplar.Exemplar {
 	return result
 }
 
+func FromHistogramProtoToHistogram(hp Histogram) *histogram.Histogram {
+	return &histogram.Histogram{
```

Comment:
What's about `CounterResetHint`? Looks like missing.

Reply:
currently Prometheus does nothing with this field, I'll ask on the sync meeting

Reply:
after discussion, opened PR on Prometheus

Reply:
updated this PR to unmarshal reset hint properly